### DISCUSSION TITLE
Adding principle to existing certificate

### DISCUSSION THREAD

**Discussion by asakth22** [gravitational/teleport#7255] [Q&A]
Using 'tctl' I created a SSH certificate and added a principle to it.
For ex, #tctl auth sign --host=x.x.x.x --format=openssh --out=ssh_cert
How can I add more servers to this existing SSH certificate ?

**Comment by webvictim** [comment]
You'll need to rerun the command:

`tctl auth sign --host=x.x.x.x,y.y.y.y --format=openssh --out=ssh_cert`

**Reply by asakth22** [reply]
I'm concerned that the existing certificate would be overwritten and then I've to copy it again to all the clients.

**Reply by webvictim** [reply]
That is correct, if you issue a new certificate for a host with an updated list of principals then you’ll need to copy it to that host.
